blog/district-c-collaboration/index.md

```diff
@@ -0,0 +1,201 @@
+documentation process and collaboration. Even a year later, I still actively use
+the good ideas I picked up during the District C project.
+
+![](images/group_photo.jpg) **Happy participants at the Hitachi Energy Office in
+Krakow.**
+
+## Cross-cultural collaboration
+
+Shortly after the project was completed, we met the US participants in the
+Krakow office and asked about their experiences.
+
+The American view by Quinn Williams:
+
+> _This program was especially helpful in understanding how different people
+> with different backgrounds can approach business issues. We got to work with
+> Polish students, and sharing ideas required us to keep our communication
+> skills at the forefront. If we had not practiced effective communication, we
+> would not have successfully conveyed some implied thoughts from our American
+> point of view. This foundation helped us understand the importance of creating
+> a strong team bond and interpersonal practices.<br></br><br></br> I also
+> enjoyed experiencing the program\'s professional side, where we communicated
+> with the Hitachi Energy team members. It was interesting to see where the
+> Polish work culture differed from the American culture and where there was an
+> overlap. I think our teams were able to use essential pieces from both sides
+> to create an effective workflow._
+
+The Polish view by Monika Dąbrowska, a participant in the technical writing
+postgraduate studies at Vistula non-public university:
+
+> _What I particularly liked about District C was that we could exchange our
+> experiences regarding learning and creating projects. The Polish education
+> system and culture lead to a general mindset somewhat contrasting to that of
+> American students._<br></br><br></br> _Also, the American view of the Poles is
+> far from what we showed during the project. Similarly, the Poles also have
+> certain misconceptions about the Americans. These assumptions and
+> presuppositions were discussed and challenged during our cooperation, and we
+> found power in our variety, creativity, exchanging experiences, and
+> understanding one another._
+
+## Kudos
+
+We are grateful to the students willing to listen to our stories and support us,
+the practitioners of the obscure art of tech writing. One can always count on
+the students to come up with fresh ideas! They were bright, open-minded, and not
+burdened with presumptions. We wish you all the best!
+
+And thank you, Dr. Cummings, for creating such a fruitful initiative — we really
+enjoy being a part of it.
+
+## The 2024 Edition
+
+In 2024, District C returned to Hitachi Energy. This time, it was led by
+Karolina Harazim, our Internal Communications Senior Specialist, and supported
+by the Tech Writing Team.
+
+Something tells me this won't be the last time Hitachi Energy collaborates with
+the University of North Carolina Wilmington. Until the next time!
+
+## More details about District C
```

Comment:
Zastsn się czy ten fragment nie powinien być na początku artykułu.

Reply:
bylo tak we wczesniejszych wersjach, pozniej stwierdzilismy ze za duzo pisalismy o samym District C na poczatku i dlatego wybralismy powiedziec nieco mniej na poczatku a dac takie odniesienie na koniec, jak ktos juz przeczytal to co na wstepie, opis przebiegu programu i chcialby na tym etapie rozeznac sie w tym i skontaktowac z organizatorem.